tests/assembly/closure-inherit-target-feature.rs

```diff
@@ -15,9 +15,8 @@ pub unsafe fn sse41_blend_nofeature(x: __m128, y: __m128) -> __m128 {
         // check that _mm_blend_ps is not being inlined into the closure
         // CHECK-LABEL: {{sse41_blend_nofeature.*closure.*:}}
         // CHECK-NOT: blendps
-        // CHECK: {{call .*_mm_blend_ps.*}}
+        // CHECK: {{jmp .*_mm_blend_ps.*}}
```

Comment:
huh, that's interesting...

Reply:
Turns out on some targets this is still `call`. I guess I'll make it accept both and remove the `ret` check?

Reply:
Actually I had a better idea: use an explicit return pointer to mimic the previously used ABI.